lib/ruby-push-notifications/apns/apns_connection.rb

```diff
@@ -26,14 +26,16 @@ class APNSConnection
       #
       # @param cert [String]. Contents of the PEM encoded certificate
       # @param sandbox [Boolean]. Whether to use the sandbox environment or not.
+      # @param optional pass [String]. Passphrase for the certificate.
+      # @param optional options [Hash]. Options for the TCP socket
```

Comment:
Options are not only for the TCP socket, it could be used for many more things later on. Also I'd add a list of currently used fields.